src/test/java/org/fundacionjala/coding/SolutionTest.java

```diff
@@ -0,0 +1,110 @@
+package org.fundacionjala.coding;
+
+//import java.util.Random;
+//import java.util.concurrent.ThreadLocalRandom;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class SolutionTest {
+    /**
+     *
+     */
+    @Test
+    public void testTwoWords() {
+        assertEquals("TestCase", Solution.camelCase("test case"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testThreeWords() {
+        assertEquals("CamelCaseMethod", Solution.camelCase("camel case method"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testLeadingSpace() {
+        assertEquals("CamelCaseWord", Solution.camelCase(" camel case word"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testTrailingSpace() {
+        assertEquals("SayHello", Solution.camelCase("say hello "));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testSingleLetter() {
+        assertEquals("Z", Solution.camelCase("z"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testTwoSpacesBetweenWords() {
+        assertEquals("AbC", Solution.camelCase("ab  c"));
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testEmptyString() {
+        assertEquals("", Solution.camelCase(""));
+    }
+
+    //private static final int NUMBER_OF_RANDOM_TESTS = 1_000;
+
+/*
+    */
+/**
+     *
+     *//*
+
+    @Test
+    public void testRandomInput() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)